test/functional/mempool_limit.py

```diff
@@ -5,70 +5,91 @@
 """Test mempool limiting together/eviction with the wallet."""
 
 from decimal import Decimal
+from test_framework.blocktools import COINBASE_MATURITY
 
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import assert_equal, assert_greater_than, assert_raises_rpc_error, create_confirmed_utxos, create_lots_of_big_transactions, gen_return_txouts
+from test_framework.util import (
+    assert_equal,
+    assert_greater_than,
+    assert_raises_rpc_error,
+    gen_return_txouts,
+)
+from test_framework.wallet import MiniWallet
+
 
 class MempoolLimitTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
-        self.extra_args = [[
-            "-acceptnonstdtxn=1",
-            "-maxmempool=5",
-            "-spendzeroconfchange=0",
-        ]]
+        self.extra_args = [
+            [
+                "-acceptnonstdtxn=1",
+                "-maxmempool=5",
+                "-spendzeroconfchange=0",
+            ]
+        ]
         self.supports_cli = False
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
-
     def run_test(self):
+
+        node = self.nodes[0]
+        miniwallet = MiniWallet(node)
+        relayfee = node.getnetworkinfo()["relayfee"]
+
+        self.log.info("Check that mempoolminfee is minrelaytxfee")
+        assert_equal(node.getmempoolinfo()["minrelaytxfee"], Decimal("0.00001000"))
+        assert_equal(node.getmempoolinfo()["mempoolminfee"], Decimal("0.00001000"))
+
+        # Generate 68 UTXOs to flood the mempool
```

Comment:
68 is no longer accurate

```suggestion
        # Generate 77 UTXOs to flood the mempool
```

this demonstrates one of the reasons it's better for comments to supplement reasoning rather than what the code is doing- if the code changes, it's easy for the comments to fall out of date.

So a better option would be to remove the hardcoding of the UTXO number from these comments.